src/app/component/MyAccount/MyAccount.component.js

```diff
@@ -235,7 +276,7 @@ class MyAccount extends Component {
                 <article block="MyAccount" elem="Additional">
                     <section aria-labelledby="forgot-password-labe">
                         <h4 id="forgot-password-label">Forgot password?</h4>
-                        <a href="#password-reset" onClick={ () => this.changeState(STATE_FORGOT_PASSWORD) }>
+                        <a href="#forgot-password" onClick={ () => this.changeState(STATE_FORGOT_PASSWORD) }>
```

Comment:
We don't need this links I suppose. Maybe we should remove them?

Reply:
Will remove them all